.github/workflows/ci.yaml

```diff
@@ -45,3 +45,17 @@ jobs:
         run: swift package resolve
       - name: Build
         run: swift build
+         
+  unit-test-ios:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Swift
+        uses: slashmo/install-swift@v0.3.0
+        with:
+          version: 5.6
+      - name: Resolve Swift dependencies
+        run: swift package resolve
```

Comment:
I'm pretty sure we can skip these two steps, since GitHub Actions' macOS runner comes with Xcode/Swift pre-installed.

```suggestion
```